mappings/net/minecraft/network/PacketByteBuf.mapping

```diff
@@ -683,3 +683,8 @@ CLASS net/minecraft/class_2540 net/minecraft/network/PacketByteBuf
 		COMMENT
 		COMMENT @return the read chunk position
 		COMMENT @see #writeChunkPos(ChunkPos)
+	METHOD method_37435 writeOptional (Ljava/util/Optional;Ljava/util/function/BiConsumer;)V
+		ARG 1 value
+		ARG 2 serializer
+	METHOD method_37436 readOptional (Ljava/util/function/Function;)Ljava/util/Optional;
```

Comment:
Mind add javadocs for this method and add the two methods to the javadoc's table of this class?